exercises/change/change_test.go

```diff
@@ -19,7 +19,7 @@ func TestChange(t *testing.T) {
 		if tc.valid {
 			if err != nil {
 				t.Fatalf("%s : Change(%v, %d): expected %v, got error %s",
-					tc.description, tc.coins, tc.target, tc.expectedChange, err)
+					tc.description, tc.coins, tc.target, tc.expectedChange, err.Error())
```

Comment:
So this is another way to ensure the 'error' type is used. It is more compact than other options like:
> var _ error = err
I like it.